atm/Sidorchuk/task15/ATM/money/Money.java

```diff
@@ -0,0 +1,48 @@
+package task15.ATM.money;
+
+public class Money {
+
+    private Nominal nominal;
+    private Currency currency;
+    private int nominalNumber;
+
+    public Money(Nominal nominal, Currency currency) {
+        this.nominal = nominal;
+        this.currency = currency;
+        nominalNumber = getNominalNumber();
+    }
+
+    private int getNominalNumber() {
```

Comment:
Look how we defined enums with private constructor. You can use your enum as ONE(1), TWO(2) and so on and move the getNominal to it in a much better and concise way.

This is, defenetely, an important point